packages/bpk-component-nudger/src/BpkNudger.tsx

```diff
@@ -16,41 +16,81 @@
  * limitations under the License.
  */
 
+import { useRef } from 'react';
+
+import { BpkButtonV2, BUTTON_TYPES } from '../../bpk-component-button';
+import { withButtonAlignment } from '../../bpk-component-icon';
+import MinusIcon from '../../bpk-component-icon/sm/minus';
+import PlusIcon from '../../bpk-component-icon/sm/plus';
 // @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import BpkLabel from '../../bpk-component-label';
 import BpkText, { TEXT_STYLES } from '../../bpk-component-text';
-import { cssModules } from '../../bpk-react-utils';
+import { cssModules, setNativeValue } from '../../bpk-react-utils';
 
-import BpkConfigurableNudger from './BpkConfigurableNudger';
 import { type CommonProps } from './common-types';
 
 import STYLES from './BpkNudger.module.scss';
 
 const getClassName = cssModules(STYLES);
 const compareValues = (a: number, b: number): number => a - b;
-const incrementValue = (currentValue: number): number => currentValue + 1;
-const decrementValue = (currentValue: number): number => currentValue - 1;
-const formatValue = (currentValue: number): string => currentValue.toString();
 
 const BpkNudger = ({
   buttonType = 'secondary',
   className = null,
+  decreaseButtonLabel,
   icon,
   id,
+  increaseButtonLabel,
+  inputClassName,
+  max,
+  min,
+  name,
+  onValueChange,
+  step = 1,
   subtitle,
   title,
+  value,
   ...rest
 }: CommonProps) => {
+  // Ensure that the minimum value is 0 or greater it's not designed to handle negative values.
+  const minimum = min < 0 ? 0 : min;
```

Comment:
With this setup it seems that once you go passed zero upwards you can never get back to zero according to this: https://backpack.github.io/storybook-prs/3576/?path=/story/bpk-component-nudger--stateful

Reply:
Hmmm that is strange I think i need to make it `<=`

Reply:
Ahh that example is set to 1 for minium. with a defualt value of 0 😂